### DISCUSSION TITLE
module 'pybop' has no attribute 'plot'

### DISCUSSION THREAD

**Discussion by Santiagopeacely** [pybop-team/PyBOP#554] [Q&A] [answered]
This  is why？ 
![image](https://github.com/user-attachments/assets/67483726-fe93-4949-bf8b-0f0d8ae0f2d1)

**Comment by BradyPlanden** [comment, marked as answer]
Hi @Santiagopeacely, we've recently changed the plotting methods in the `develop` branch, which will released in the next version. The examples in the `develop` branch won't work with PyBOP v24.9.1 or previous as this change is breaking. For the examples that work with v24.9.1, follow these [examples](https://github.com/pybop-team/PyBOP/tree/main/examples) (which on on the `main` branch) until the next PyBOP release.

**Reply by bipcode** [reply]
I couldn't run the examples of the main branch too. Has there been any change in the main branch?

**Comment by Santiagopeacely** [comment]
That‘s great! Thank you
